classes/class3/hdavidzhu_class3_homework/routes/cat-router.js

```diff
@@ -0,0 +1,101 @@
+var faker = require('faker');
+
+function shuffle(array) {
+  var currentIndex = array.length, temporaryValue, randomIndex ;

```

Comment:
Please don't define variables this way. It is very unclear what is actually being done. Use a new line with a `var` for each variable.